lego/apps/meetings/tasks.py

```diff
@@ -0,0 +1,40 @@
+from datetime import timedelta
+
+from django.utils import timezone
+
+from structlog import get_logger
+
+from lego import celery_app
+from lego.apps.meetings import constants
+from lego.apps.meetings.models import Meeting, MeetingInvitation
+from lego.apps.meetings.notifications import MeetingReminderNotification
+from lego.utils.tasks import AbakusTask
+
+log = get_logger()
+
+
+@celery_app.task(serializer="json", bind=True, base=AbakusTask)
+def notify_user_of_unanswered_meeting_invitation(self, logger_context=None):
+    self.setup_logger(logger_context)
+
+    meetings: list[Meeting] = Meeting.objects.filter(
+        start_time__gt=timezone.now()
+    ).prefetch_related("registrations")
+
+    for meeting in meetings:
+        # Only send reminder after 2 days
+        if meeting.created_at > timezone.now() - timedelta(days=2):
+            continue
```

Comment:
You can do this check in the query :) Just like with the start time check.

e.g.
```python
Meeting.objects.filter(created_at__lt=timezone.now() - timedelta(days=2))
```

Reply:
Ah yes of course, that's cleaner! I'm assuming you mean `created_at__gt` instead of `created_at__lt` though 👍🏼

Reply:
Shouldn't it be lt? A timestamp from last year would be less(smaller number) than a timestamp from two days ago, while todays timestamp would be greater than

Reply:
Oh shoot you're right, I was simply looking at the if-statement that I so carefully constructed yesterday, but when filtering the query, it's the other way around. Sorry!